releases/evm/2025-01-ITS-v2.1.0.md

```diff
@@ -118,8 +124,15 @@ node evm/interchainTokenFactory.js --action linkToken --destinationChain chain2
 node evm/its.js --action transferMintership --tokenAddress [tokenAddress] --minter [tokenManager]
 ```
 
+> [!NOTE]  
+> `tokenManager` address is different per token, which can be found on tx output from `registerCustomToken` action on chain
```

Comment:
This should be before the cmd that needs the token manager not after someone runs it incorrectly. Also, instead of a note, show how to use `evm/its.js --action tokenManagerAddress ...` to get the address on each chain as a step